pkg/api/testing/compat/compatibility_tester.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 // Based on: https://github.com/openshift/origin/blob/master/pkg/api/compatibility_test.go
-//
+
```

Comment:
Don't disconnect this section, just move it below the other part of the comment.

Reply:
Fixed